service/controller/machinepool/handler/migration/testdata/new_machinepool_references_not_updated.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Test input, new MachinePool CR that is not fully updated (references to other CRs are not yet updated).